code/game/jobs/job/job.dm

```diff
@@ -248,10 +248,10 @@
 		C.sex = capitalize(H.gender)
 		C.age = H.age
 		C.name = "[C.registered_name]'s ID Card ([C.assignment])"
-		C.photo = get_id_photo(H)
-
-		if(H.mind && H.mind.initial_account)
-			C.associated_account_number = H.mind.initial_account.account_number
+		if(H.mind)//avoid mindless mob errors with these procs

```

Comment:
so pretty much i looked in imprint ID (We have TG dosen't causes runtimes on our end) decided just to split up and scoot these over into a general mind check. seems to fix the issue.